packages/uui-css/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@umbraco-ui/uui-css",
+  "version": "0.0.0",
+  "license": "MIT",
+  "keywords": [
+    "Umbraco",
+    "Custom elements",
+    "Web components",
+    "UI",
+    "Lit",
+    "Css"
+  ],
+  "description": "Umbraco UI css component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/umbraco/Umbraco.UI.git",
+    "directory": "packages/uui-css"
+  },
+  "bugs": {
+    "url": "https://github.com/umbraco/Umbraco.UI/issues"
+  },
+  "main": "./dist/uui-css.min.js",
+  "module": "./lib/index.js",
+  "customElements": "custom-elements.json",
+  "files": [
+    "dist",
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "custom-elements.json"
+  ],
+  "dependencies": {
+    "lit": "^2.0.0"
```

Comment:
Why do we have a dependency on lit in this package? Wouldn't it be better to depend on the base package so we always used the same version of lit?